### PR TITLE
data :AzraNaseri

### PR DESCRIPTION
**Pull request by AzraNaseri**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/AzraNaseri.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/AzraNaseri",
+      "icon": "FaGithub"
+      
+    }
+  ],
+  "social": [
+    {
+      "name": "Follow me on LinkedIn",
```
Comment (kumarsonsoff3) [inline data/AzraNaseri.json:14]: Socials don't require a `name` key. Also there is a missing `s` in `"socials"`

```suggestion
  "socials": [
    {
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/AzraNaseri.json (hunk 1 above)
Comment (AzraNaseri) [reply data/AzraNaseri.json]: I updated the codes and implemented what you mentioned. So why has it still not merged?

Comment (AzraNaseri) [reply data/AzraNaseri.json]: and how can i open my codes in the browser i mean to see the result (profile)?

Comment (kumarsonsoff3) [reply data/AzraNaseri.json]: You'll find your profile at https://biodrop.io/AzraNaseri after this PR is being merged and deployed!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉

**Pull request merged**